libr/asm/arch/jdh8/jdh8dis.c

```diff
@@ -0,0 +1,140 @@
+/* radare - LGPL3 - Copyright 2021 - condret, slowhand99 */
+
+#include <r_types.h>
+#include <r_util.h>
+#include <r_asm.h>
+
+static const int jdh_len[16] = {
+	2, 3, 3, 1, 1, 3, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2
+};
+
+static const char reg[6] = { 'A', 'B', 'C', 'D', 'L', 'H' };
+
+static int jdh_get_ins_len(ut8 hex) {
+	ut8 high = (hex & 0xf0) >> 4;
+	int ret = jdh_len[high];
+	if (ret == 3) {
+		ret = (hex & 8) ? 2 : 3;
+	}
+	if (ret == 1) {
+		ret = (hex & 8) ? 1 : 2;
+	}
+	return ret;
+}
+
+static int jdh8Disass(RAsmOp *op, const ut8 *buf, int len) {
+	int rlen = jdh_get_ins_len (*buf);
```

Comment:
why `rlen`? maybe `ilen` works better as a name as it stands for `instruction length`